the_debuginator.h

```diff
@@ -1304,11 +1317,38 @@ bool debuginator__distance_to_hot_item(DebuginatorItem* item, DebuginatorItem* h
 
 			child = debuginator__next_visible_sibling(child);
 		}
+	} else if (item->leaf.is_expanded && !item->is_filtered) {
+		*distance += item_height * item->leaf.num_values;
+		*distance += item->leaf.description_line_count * item_height;
 	}
-
 	return false;
 }
 
+int debuginator_total_height(TheDebuginator* debuginator) {
```

Comment:
This doesn't seem to get called from this commit. I assume it's needed by something in one of the other PRs? 

Couldn't you simply return total_height for the root item? If not, what's the difference?